pkg/query-service/app/logs/parser.go

```diff
@@ -304,6 +304,13 @@ func replaceFieldInToken(queryToken string, selectedFieldsLookup map[string]mode
 			} else if strings.Compare(strings.ToLower(*col), "fulltext") != 0 && field.Type != constants.Static {
 				return "", fmt.Errorf("field not found for filtering")
 			}
+		} else {
+			field := selectedFieldsLookup[sqlColName]
+			if field.Type != constants.Static {
+				prefix := field.Type[:len(field.Type)-1]
+				// columns name is <type>_<name>_<datatype>
+				sqlColName = fmt.Sprintf("%s_%s_%s", strings.ToLower(prefix), strings.ToLower(field.DataType), sqlColName)
+			}
```

Comment:
Couldn't follow this. Please help me understand this.

Reply:
So in the previous logs parser if we send a query as `method gt 200`

the backend first extracts the name `method` and check if it is an selected field, if not then it converts to the correct attribute name i.e `attributes_int64_vale[indexOf(attributes_in64_key, 'method'')]` for SQL query

But if it is a selected field then it add `method > 200` in the SQL query which is changed with recent changes where it should be `attributes_int64_method > 200`